addons/api/package.json

```diff
@@ -49,7 +49,7 @@
     "concurrently": "^8.0.1",
     "doctoc": "^2.2.0",
     "ember-cli": "^4.12.2",
-    "ember-cli-dependency-checker": "^3.3.1",
```

Comment:
upgrade to `ember-cli-dependency-checker` doesn't fix anything since downstream braces v2 is still used but thought we should update either way